README.rst

```diff
@@ -32,3 +32,10 @@ coala supports popular programming languages including **Python**, **C/C++**, **
    :target: http://docs.coala.io/
 .. |AGPL| image:: https://img.shields.io/pypi/l/coala.svg
    :target: https://www.gnu.org/licenses/agpl-3.0.html
+
+
+.. image:: https://graphs.waffle.io/coala/coala/throughput.svg
+ :target: https://waffle.io/coala/coala/metrics/throughput
+ :alt: 'Throughput Graph'
+
+*Last updated: 27th April, 2017*
```

Comment:
I initially thought we have to update it manually. Right now I tried it on one of my repos and the image updates by itself.